wecs/mechanics/clock.py

```diff
@@ -93,20 +96,24 @@ class UpdateCalendar(System):
         ]),
     }
 
+    def increment_timeframes(self, calendar, time_since):
+        timeframes = list(calendar.timeframes.keys())
+        calendar.timeframes[timeframes[0]][0]+=time_since
+        for t, timeframe in enumerate(timeframes):
+            if t < len(timeframes)-1:
```

Comment:
If you don't want to include the last timeframe, you can also just use `enumerate(timeframes[:-1])`